src/app/sections/payment-routing-rules/routing-rules-select/routing-rules-select.component.ts

```diff
@@ -0,0 +1,8 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+    selector: 'cc-routing-rules-select',
+    templateUrl: 'routing-rules-select.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class RoutingRulesSelectComponent {}
```

Comment:
Component for what? It's a ng-content only

Reply:
del